app/src/screens/Dialer/index.tsx

```diff
@@ -8,52 +8,44 @@ import OutgoingRemoteParticipant from './OutgoingRemoteParticipant';
 import useDialer from './hooks';
 
 const Dialer: React.FC = () => {
-  const {
-    dialpad,
-    isDisabled,
-    outgoingRemoteParticipant,
-    makeOutgoingCall,
-    recipient,
-  } = useDialer();
+  const { dialpad, makeCall, outgoing, recipientToggle } = useDialer();
 
-  const isOutgoingCallButtonDisabled =
-    dialpad.outgoingNumber === '' || isDisabled;
-
-  const isBackSpaceVisible = dialpad.outgoingNumber !== '';
+  const backspaceButton = React.useMemo(
```

Comment:
nit: since this is a component lets capitalize it
```suggestion
  const BackspaceButton = React.useMemo(
```

Reply:
I actually think it looks better capitalized, but I'm not sure if it's technically proper since it's not _really_ a React component, it's an instantiated component and therefore a `JSX.Element`. Minor difference, but there's implications since it's not a "constructor" and it's a "instance" basically. Let me look up the recommended practices.

Reply:
I couldn't find anything concrete on this one, @charliesantos @kpchoy what do you think? I'm open to either, but the current codebase is lowercased.

Reply:
The convention is not set in stone. But historically, we use `PascalCase` for exported components, `camelCase` otherwise.

Reply:
Just thought it looks better, but lets leave it then

Reply:
I don't have any preference. We can change the convention. But we need to make sure the whole codebase follows for consistency. We cannot change only in one place. If we like `PascalCase` better, please add a ticket to change this on a future sprint :)